packages/rainbow-button/CHANGELOG.md

```diff
@@ -16,17 +16,17 @@
 
   **2. Configure with Wagmi and install the RainbowButton**
 
-  Pass an instance of the `RainbowConnector` to your Wagmi connector list, and wrap your app in the `RainbowButtonProvider`. Then drop-in the `RainbowButton` component into your wallet list.
+  Pass an instance of the `rainbowConnector` to your Wagmi connector list, and wrap your app in the `RainbowButtonProvider`. Then drop-in the `RainbowButton` component into your wallet list.
 
   ```tsx
   import "@rainbow-me/rainbow-button/styles.css";
   import {
     RainbowButtonProvider,
-    RainbowConnector,
+    rainbowConnector,
   } from "@rainbow-me/rainbow-button";
 
   const wagmiClient = createConfig({
-    connectors: [new RainbowConnector({ chains, projectId })],
+    connectors: [rainbowConnector({ projectId })],
     publicClient,
   });
 
```

Comment:
Wagmi V2 no longer uses classes like `new InjectedConnector()`. Instead they have pure functions like `walletConnect()` or `ledger()`. Want to keep it similar to what they have 🙏